### PR TITLE
added music player game

### LINKED ISSUE #56: Music Player
## Is your feature request related to a problem? Please describe.
Yes
## Describe the solution you'd like
Would like to make a javascript music player game made using html, css, js

## Additional context
Add any other context about the feature request here.
It wiil be having a good background animation

### PR DESCRIPTION
**Pull request by Sloth-Panda**
#### Issue Number
fixes #56 

#### Describe the changes you've made
A clear and concise description of what you have done to successfully close your assigned issue. Any new files? or anything you feel to let us know!
This is a music player with cool animation made using html, css, and vanilla JS. On striking each button you will be getting their respective musics.
#### Describe if there is any unusual behaviour of your code(Write `NA` if there isn't)
A clear and concise description of it.
No unusual behaviour
#### Additional context (OPTIONAL)
game2

#### Test plan (OPTIONAL)
A good test plan should give instructions that someone else can easily follow.
How someone can test your code?
https://sloth-panda.github.io/MusicPlayer/ here's the link.
#### Checklist

- [ ] I have added my name in the contributors list at the end of README.md file.
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.
- [x] The title of my pull request is a short description of the requested changes.
- [ ] Open Source Program names(OPTIONAL: If you participated in any open-source program then please mention the program name here)

### CODE REVIEW

File: assets/css/musicplayer.css [hunk 1]
```diff
@@ -0,0 +1,102 @@
+*{
+    padding: 0;
+    margin: 0;
+    box-sizing: border-box;
+}
+
+body{
+    font-family: 'Space Grotesk', sans-serif;
+}
+
```
Comment (Vishal-raj-1) [inline assets/css/musicplayer.css:10]: There should be only one line gap. Remove extra line !!

File: assets/css/musicplayer.css [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+    font-size: 19px;
+    color: grey;
+    
+}
+
+.pads{
+    background: lightblue;
+    width: 100vw;
+    display: flex;
+
```
Comment (Vishal-raj-1) [inline assets/css/musicplayer.css:40]: There should not be any line gap !!

File: assets/css/musicplayer.css [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+.pads > div{
+    height: 150px;
+    width: 100px;
+    flex: 1;
+}
+
+.pad-1{
+    background-color: rgb(119, 211, 26);
+
```
Comment (Vishal-raj-1) [inline assets/css/musicplayer.css:51]: There should not be any line gap in all .pad-{1,2,3,4,5}!!

**Review by Vishal-raj-1 — CHANGES_REQUESTED** [review]
@Sloth-Panda Good work !! Projects seems awesome to me, just focus on code formatting as well.

**Comment by Sloth-Panda** [comment]
@Vishal-raj-1 @urvashi-code1255  is it okay now?

I actually resolved the conflicts here on github and made that commit but that was just the conflict in index page which I resolved.
After that I took a pull of code from upstream/main and their made the required changes that you mentioned  but when I am trying to push the code to it shows like this even I tried pull then push still shows like this to me
Screenshot (123)

so what to do ?

**Comment by Vishal-raj-1** [comment]
did you commit your changes  ? first commit your changes then do `git pull upstream master` not `git pull` then push.

**Comment by Sloth-Panda** [comment]
@Vishal-raj-1 I am still not able to push, I have updated my repo with the upstream but still not able to push why's that??

Screenshot (127)

**Comment by Sloth-Panda** [comment]
@Vishal-raj-1 bro, I checked out a new branch and pushed the code to that branch and the push worked also every other thing works fine so for these issues can I make a single separate pr? and close these two separate prs?

**Comment by Sloth-Panda** [comment]
@urvashi-code1255, I have some doubts (1). I see when I am making one pr, it is having commits of the previous prs I mean how can  I make commits for only one issue in one pr? Like here you saw what should have been 3 prs commits got merged in 1pr?
(2) how can I get this site or sites like this to work https://Sloth-Panda.github.io/

**Comment by urvashi-code1255** [comment]
> @urvashi-code1255, I have some doubts (1). I see when I am making one pr, it is having commits of the previous prs I mean how can I make commits for only one issue in one pr? Like here you saw what should have been 3 prs commits got merged in 1pr?

@Sloth-Panda To make PRs with different changes, you should make all the PRs from seperate branches
> (2) how can I get this site or sites like this to work https://Sloth-Panda.github.io/

I didn't get your point!!

**Pull request closed without merging**